### ISSUE TITLE
Change music with touch bar out of YT music

### ISSUE THREAD

**Issue by jo541** [steve228uk/YouTube-Music#53]
I don't know if it normally or I have bad config. But when I m on other application in full screen I cannot change music. I have only the duration of music and play/pause.

**Comment by vveliev-tc** [comment]
+1 , not able to control songs from touchbar

this is the keys that are expected to work 
```
eventType:key_down        code:0xb3       name:fastforward     misc:
eventType:key_up          code:0xb3       name:fastforward     misc:
eventType:key_down        code:0xb3       name:fastforward     misc:
eventType:key_up          code:0xb3       name:fastforward     misc:
eventType:key_down        code:0xb3       name:fastforward     misc:
eventType:key_up          code:0xb3       name:fastforward     misc:
eventType:key_down        code:0xb4       name:rewind          misc:
eventType:key_up          code:0xb4       name:rewind          misc:
```

**Comment by nashihu** [comment]
> +1 , not able to control songs from touchbar
> 
> this is the keys that are expected to work
> 
> ```
> eventType:key_down        code:0xb3       name:fastforward     misc:
> eventType:key_up          code:0xb3       name:fastforward     misc:
> eventType:key_down        code:0xb3       name:fastforward     misc:
> eventType:key_up          code:0xb3       name:fastforward     misc:
> eventType:key_down        code:0xb3       name:fastforward     misc:
> eventType:key_up          code:0xb3       name:fastforward     misc:
> eventType:key_down        code:0xb4       name:rewind          misc:
> eventType:key_up          code:0xb4       name:rewind          misc:
> ```

+1

any update please?

**Comment by j-frost** [comment]
I agree this would be a great addition. Considering Google have made Chrome apps "installable", maybe they'll add media key support in the future?
